rune/go.mod

```diff
@@ -3,27 +3,35 @@ module github.com/inclavare-containers/rune
 go 1.14
 
 require (
+	github.com/alibaba/inclavare-containers/epm v0.0.0-20201022145650-08a6ceb72088
+	github.com/alibaba/inclavare-containers/rune v0.0.0-00010101000000-000000000000
```

Comment:
`rune` declares itself `module github.com/inclavare-containers/rune`, so you need to require `github.com/inclavare-containers/rune` rather than `github.com/alibaba/inclavare-containers/rune`,  and you can `replace github.com/inclavare-containers/rune => github.com/alibaba/inclavare-containers/rune v0.0.0-20201022105457-b72af047bfaa` just like [go.mod in sgx-tools](https://github.com/alibaba/inclavare-containers/blob/master/sgx-tools/go.mod)

Reply:
`rune` changes frequently, so you should specify a specific commit id just like `b72af047bfaa` and its corresponding time like `20201022105457`.  Finally, you should write it in go.mod in the form of `github.com/inclavare-containers/rune => github.com/alibaba/inclavare-containers/rune v0.0.0-20201022105457-b72af047bfa`